main_spec.tex

```diff
@@ -116,6 +116,83 @@ \subsubsection{\textbf{SHPDEALLC}}\label{subsec:shpdeallc}
 \input{content/shpdeallc.tex}
 
 
+
+
+\color{Green}
+\subsection{Teams Management Routines}\label{subsec:team}
+
+The \acp{PE} in an \openshmem program can communicate either using
+point-to-point routines that specify the \ac{PE} number of the target
+\ac{PE} or using collective routines which operate over some predefined
+set of \acp{PE}. Teams in \openshmem allow programs to group subsets
+of \acp{PE} for collective communications and provide a contiguous renumbering
```

Comment:
renumbering -> reindexing ?

Reply:
I had been using the term "number" and "numbering" throughout to replace "rank". But "reindex" and "indexing" are more clear terms than "renumber" and "numbering". I think using "PE number" to mean similar to "MPI rank" and then using the term indexing for the verb form sounds better. So I will switch to using that.